ci/pkl-pipelines/pay-deploy/pay-cli.pkl

```diff
@@ -0,0 +1,45 @@
+amends "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.4#/Pipeline.pkl"
+
+import "../common/pipeline_self_update.pkl"
+import "../common/shared_resources.pkl"
+import "../common/PayResources.pkl"
+
+resources = new {
+  pipeline_self_update.PayPipelineSelfUpdateResource("pay-deploy/pay-cli.pkl", "master")
+  shared_resources.payCiGitHubResource
+  new PayResources.PayGitHubResource {
+    name = "pay-cli-git-main"
+    repoName = "pay-cli"
+    branchName = "main"
+    commitFilter = new {
+      exclude = new {"\\[automated release\\]"}
+    }
+  }
+}
+
+jobs {
+  pipeline_self_update.PayPipelineSelfUpdateJob("pay-deploy/pay-cli.pkl")
+  new {
+    name = "version-and-push"
+    plan {
+      new InParallelStep {
+        in_parallel = new Listing<Step> {
+          new GetStep {get = "pay-ci"}
```

Comment:
I'm willing to be told this is heresy to one-line simple objects, I find it simpler to read than scanning multiple lines. I see in the pkl style guide info about nested objects being indented but nothing about not one-lining simple object creation...